engine/src/main/java/org/pentaho/di/core/LastUsedFile.java

```diff
@@ -49,6 +49,12 @@ public class LastUsedFile {
   private String repositoryName;
   private Date lastOpened;
   private String username;
+  /**
+   * Separate VFS connection name variable is no longer needed.
+   * @deprecated
+   * The connection name is in the URI since full {@value org.pentaho.di.connections.vfs.provider.ConnectionFileProvider#SCHEME } paths are being used.
+   */
```

Comment:
Adding deprecated to all instances of the user of variables `connection` and `domain`, cleanup effort will is tracked by https://hv-eng.atlassian.net/browse/BACKLOG-40656